modules/core/src/worker-utils/get-transfer-list.js

```diff
@@ -0,0 +1,39 @@
+// NOTE - there is a copy of this function is both in core and loader-utils
```

Comment:
Can import from `loader-utils`?

Reply:
Of course, although this will bloat core even further as many of the loader utils are only used by some loaders. Will consider that separately.